### DISCUSSION TITLE
Not enough margin

### DISCUSSION THREAD

**Discussion by ndharris** [tinymce/tinymce#9600] [General]
Hi there,

Recently we've noticed that there isn't enough margin in the editing pane, meaning you can't see all of the first letter of each line.

![image](https://github.com/tinymce/tinymce/assets/6492991/3d8f3b2c-3280-4b68-a36f-ff65506d4c43)

I hope you can add a bit of margin to make it look nicer.

Thanks!

Neil.

**Comment by TheSpyder** [comment]
Your editor has custom content CSS, the margins are likely set the same way it makes the background yellow. TinyMCE doesn't have margins like that by default, for example:
https://fiddle.tiny.cloud/

**Comment by ndharris** [comment]
Ah yes, thank you very much, I should have spotted that.  My website's body CSS has zero margin and this was being inherited into the text pane of the editor.

I think something has changed in the editor because it didn't used to do that.  I have added the following CSS to my stylesheet to resolve this:

```
body#tinymce {
        margin:8px;
}
```

**Reply by TheSpyder** [reply]
Have you upgraded the editor recently? which version did you use before? Perhaps one of our old default styles was overriding your default and that no longer happens.

**Comment by ndharris** [comment]
Yes, upgraded a few weeks ago, and you are probably right.  Thanks for your time.
